conda-recipes/impyla/meta.yaml

```diff
@@ -25,6 +26,7 @@ requirements:
     - thrift_sasl
     - bitarray
     - thriftpy
+    - sasl >=0.2.1
```

Comment:
I will get it.

On Thursday, March 24, 2016, Wes McKinney <notifications@github.com> wrote:

> In conda-recipes/impyla/meta.yaml
> <https://github.com/cloudera/ibis/pull/805#discussion_r57403207>:
>
> > @@ -25,6 +26,7 @@ requirements:
> >      - thrift_sasl
> >      - bitarray
> >      - thriftpy
> > +    - sasl >=0.2.1
>
> Sure, let's change that and I can make a minor release. Can someone make a
> PR?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudera/ibis/pull/805/files/0d0752a52b6ef4d6bc42b36b6c599f12d937f785#r57403207>
>